packages/app-project/test/fixtures/project.js

```diff
@@ -0,0 +1,18 @@
+const project = {
```

Comment:
I have project mocks I setup for the tests in the panoptes.js client that looks like I forgot to export in the lib at the top level. Would you like to reuse those for tests?

Reply:
Absolutely yes <3

Reply:
See #32